Makefile

```diff
@@ -40,16 +44,15 @@ check-reqs:
 shellcheck:
 	$(ROOT_DIR)/tools/shellcheck -e SC1091 jenkins-support *.sh
 
-build:
+build: check-reqs
```

Comment:
not sure if you intentionally left out `check-reqs` here, I added it as it makes sure the buildx plugin is installed

Reply:
Sounds like a mistake from me. Good catch 👍